src/basic_recipes/axis.jl

```diff
@@ -538,6 +538,8 @@ end
 function plot!(scene::SceneLike, ::Type{<: Axis2D}, attributes::Attributes, args...)
     # create "empty" plot type - empty meaning containing no plots, just attributes + arguments
     cplot = Axis2D(scene, attributes, args)
+    # Disable any non linear transform for the axis plot!
```

Comment:
I suppose we should override these with different axis implementations?  For example, a generic axis which works across all transforms only requires you to densely sample your lines, such that they look smooth when plotted.